code/modules/modular_computers/hardware/battery_module.dm

```diff
@@ -59,7 +59,7 @@
 			user.put_in_hands(battery)
 			to_chat(user, span_notice("You detach \the [battery] from \the [src]."))
 		else
-			battery.forceMove(drop_location())
+			QDEL_NULL(battery) //Skyrat Edit Start - These batteries are outrageous
```

Comment:
Concerned this might just qdel any batteries that are ejected for other reasons, if there's no user for whatever reason